### PR TITLE
Project 2 News site - Josephine Lam

### PR DESCRIPTION
**Pull request by prosperpine**
Project two, News site with responsive design using flexbox. 
I did the design as required using media queries and a couple of the stretch goals, ie hover-effects and box-shadow. 

I struggled some with making the different images fit. And when I added links, a href, to the different sections, the margins and/or padding changed and moved things around. Hence only the big-news-section and the nav bar contain links. 

Even though the layout changes as planned in desktop when I change the size of the browser, it didn't look as planned (with margins and paddings etc) when I looked at the site on my mobile. 

/Josephine

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,107 @@
...
+    <h1>the meditator</h1>
+    <h3>The number one spot for your inner life</h3>
+    <nav>
+      <a href="https://www.yogajournal.com/" target="_blank">Latest</a>
+      <a href="https://www.yogajournal.com/" target="_blank">About</a>
+      <a href="https://www.yogajournal.com/" target="_blank">Subscribe</a>
+    </nav>
+
+
+
```
Comment (ericaglimsholt) [inline code/index.html]: You can delete this empty lines

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,107 @@
...
+    </div>
+    <div class="other-news">
+
+      <div class="other-news-img">
+        <img src="meditate.jpg" alt="a person meditating in front of lake">
+      </div>
+      <div class="other-news-text">
+        <h4> Sadhguru On Why Meditation Doesn't Work For Everyone</h4>
+        <p> The moment we utter the word “meditation,” there are all kinds of misconceptions about it. First of all,
+          the
```
Comment (ericaglimsholt) [inline code/index.html]: How come "the" is on a seperate line? :)

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,271 @@
...
 }
+
+
+h1 {
+  font-family: 'Merriweather', serif;
+  font-size: 66px;
+  padding: 0;
+  text-align: center;
+  margin-bottom: 0px;
+  /**text-shadow: 2px 2px 4px #c4def0**/
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,271 @@
...
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,271 @@
...
+}
+
+h2 {
+  font-family: 'Enriqueta', serif;
+  font-size: 48px;
+  font-style: italic;
+  margin: 0px;
+  color: #0f4c75;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,271 @@
...
+h4 {
+  font-family: 'Baskervville', serif;
+  margin-top: 0;
+  margin-bottom: 14px;
+  font-size: 26px;
+  display: flex;
+  line-height: 1;
+  color: #0f4c75;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,271 @@
...
+}
+
+h5 {
+  font-family: 'Enriqueta', serif;
+  font-size: 46px;
+  text-decoration-line: underline;
+  margin: 0px;
+  color: rgb(110, 7, 173);
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,271 @@
...
+  background-image: url('header.jpg');
+  background-size: cover;
+  background-color: #0f4c75;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  background-repeat: no-repeat;
+
+
```
Comment (ericaglimsholt) [inline code/style.css]: Delete lines

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,271 @@
...
+  text-decoration: none;
+  color: white;
+  font-family: 'Nunito', sans-serif;
+  font-size: 14px;
+  padding: 16px
+}
+
+a:hover {
+  color: aqua;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 10]
```diff
@@ -1,4 +1,271 @@
...
+  display: flex;
+
+  width: 100%;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column-reverse;
+  line-height: 1, 5rem;
+  border: solid #1e6a9c 3px;
+
+
```
Comment (ericaglimsholt) [inline code/style.css]: Delete lines

File: code/style.css [hunk 11]
```diff
@@ -1,4 +1,271 @@
...
+.big-news-img {
+  max-width: 80%;
+  padding: 20px 0;
+}
+
+.big-news-text {
+  padding: 36px 36px 0px 36px;
+  line-height: 1;
+  color: #0f4c75;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 12]
```diff
@@ -1,4 +1,271 @@
...
+}
+
+.big-news-text:hover {
+  background-color: white;
+  opacity: 0.5;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 13]
```diff
@@ -1,4 +1,271 @@
...
+.other-news-section {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  flex-wrap: wrap;
+  flex-direction: column;
+  width: 100%;
+  margin: 0px;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 14]
```diff
@@ -1,4 +1,271 @@
...
+}
+
+.other-news {
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css [hunk 15]
```diff
@@ -1,4 +1,271 @@
...
+.other-news-img {
+  max-width: 100%;
+}
+
+img {
+  max-width: 100%;
+}
+
+
```
Comment (ericaglimsholt) [inline code/style.css]: Delete lines

File: code/style.css [hunk 16]
```diff
@@ -1,4 +1,271 @@
...
+}
+
+footer {
+  height: 100px;
+  background-color: rgb(185, 184, 184);
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 20px;
+
```
Comment (ericaglimsholt) [inline code/style.css]:
```suggestion
```

File: code/style.css (hunk 9 above)
Comment (prosperpine) [reply code/style.css]: Why?

**Pull request closed without merging**